package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-bio",
-  "version": "0.5.0",
+  "version": "0.5.2",
```

Comment:
Presumably this will get sorted out at publishing time, but to be clear this should be 0.6.0 as it's adding a new component.

Reply:
Thanks for pointing it out, fixed in dff2f6c.